### PR TITLE
use single info point that is extendable

### PR DESCRIPTION
**Pull request by moovida**
New PR to reflect minimal changes as discussed in previous PR.

### CODE REVIEW

File: lib/map_elevation.dart [hunk 1]
```diff
@@ -248,9 +246,9 @@ class _ElevationPainter extends CustomPainter {
 /// [Notification] emitted when graph is hovered
 class ElevationHoverNotification extends Notification {
   /// Hovered point coordinates
-  final lg.LatLng position;
+  final ElevationPoint elevationPoint;
```
Comment (tuarrep) [inline lib/map_elevation.dart]: BC here. Just let it named `position`

File: lib/map_elevation.dart [hunk 2]
```diff
@@ -269,12 +267,10 @@ class ElevationGradientColors {
 }
 
 /// Geographic point with elevation
-class ElevationPoint {
-  /// Latitude and Longitude
-  lg.LatLng latLng;
```
Comment (tuarrep) [inline lib/map_elevation.dart:274]: Please add a getter for backward compatibility to retrieve an instance of `LatLng` (basically `this`)

**Review by tuarrep — CHANGES_REQUESTED** [review]

**Comment by moovida** [comment]
@tuarrep added the requested changes. Thanks for reviewing.

**Review by tuarrep — APPROVED** [review]
LGTM

**Review by HugoHeneault — DISMISSED** [review]
Hi! Cool PR. Thanks a lot!

Could you remove `import 'package:latlong/latlong.dart';` from `example/data.dart`? It is now unused.

With that change, LGTM 👍 

Thanks

**Review by tuarrep — CHANGES_REQUESTED** [review]
Could you please rebase on master too. We've added some CI

**Comment by moovida** [comment]
Import removed and rebase done. Thanks guys!

**Review by tuarrep — APPROVED** [review]

**Pull request merged**